CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CIReinforcementsManager.uc

```diff
@@ -0,0 +1,122 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: Manager class to control the delay of any reinforcements issued
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class XComGameState_CIReinforcementsManager extends XComGameState_BaseObject;
+
+var array<DelayedReinforcementSpawner> DelayedReinforcementSpawners;
+
+var int NextReinforcementsArrival;
+var int LastTurnModified;
```

Comment:
* **`NextReinforcementsArrival`** should be a function.
* **`LastTurnModified`** - there is no need for it. The manager should always be in correct state.

What you've done is known as "multiple sources of truth". This means that you've inflicted the need for more work to keep them synchronized. You should aim to never do this (except in some rare exceptional cases, which this one is not)